src/OodDesign/CLassroom.java

```diff
@@ -0,0 +1,131 @@
+package OodDesign;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CLassroom {
+    private int number;
+    private Obj obj;
+    private Student[] students;
+
+
+
+    CLassroom(int number, Obj obj, Student[] students) {
+        this.number = number;
+        this.obj = obj;
+        this.students = students;
+    }
+
+    boolean addStudent(Student student) {
+        for (int i = 0; i < students.length; i++) {
+            if (students[i] == null) {
+                student.id = i + 1;
+                return true;
+            }
+        }
+        return false;
+    }
+
+    boolean deleteStudent(Student student) {
+        for (int i = 0; i <students.length; i++) {
+            if (students[i].equals(student)) {
+                students[i] = null;
+                return true;
+            }
+        }
+        return false;
+    }
+
+    boolean replace(Student target, Student replace) {
+        for (int i = 0; i < students.length; i++) {
+            if (students[i].equals(target)) {
+                students[i] = replace;
+                return true;
+            }
+        }
+        return false;
+    }
+
+
+    boolean find(int id) {
+        for (Student student1 : students) {
+            if (student1.id == id) {
+                return true;
+            }
+        }
+        return false;
+    }
+//重载 定义相同方法，参数不一样
+    boolean find(String fullName) {
+        for (Student student1 : students) {
+            if (student1.fullName().equals(fullName)) {
+                return true;
+            }
+        }
+        return false;
+    }
+}
+
+class Teacher {
```

Comment:
About class Teacher and student, if you set private instance variable, AND they can be visited, e.g. why someone cannot get `gender` of teacher or student? 
Then you should have setter or getter or both for it